diff --git a/build/sharp.js b/build/sharp.js
index 8504ded81e0eaea82f9f96ed38366ff193a51a98..7d327496d52cbe3026f84c95dbd9967750623e46 100644
--- a/build/sharp.js
+++ b/build/sharp.js
@@ -130,7 +130,7 @@ async function findSharpBinAsync() {
         if (sharpCliPackagePath &&
             semver_1.default.satisfies(sharpCliPackage.version, SHARP_REQUIRED_VERSION) &&
             typeof sharpCliPackage.bin.sharp === 'string' &&
-            typeof _sharpInstance?.versions?.vips === 'string') {
+            typeof sharpInstance?.versions?.vips === 'string') {
             _sharpBin = path_1.default.join(path_1.default.dirname(sharpCliPackagePath), sharpCliPackage.bin.sharp);
             _sharpInstance = sharpInstance;
             return _sharpBin;
